lib/api/helpers.js

```diff
@@ -367,3 +367,29 @@ export function convertWeight(from, to, weight) {
   // if we made it here, something has gone wrong
   throw new Meteor.Error("invalid-parameter", "Invalid from or to value specified");
 }
+
+/**
+ * @name luhnValid
```

Comment:
I wonder if we shouldn't namespace some of these helpers? What do you think @jshimko?

Reply:
Sure.  @spencern, care what that namespace is?

Reply:
@jshimko what would you suggest?

Reply:
I haven't thought about this much, open to suggestions.

Reply:
@zenweasel is this a blocker for this PR? If so can you "request changes"